### DISCUSSION TITLE
Lazer notifies me of old DM

### DISCUSSION THREAD

**Discussion by msmcculley** [ppy/osu#24756] [Q&A]
Simply put, was on the 20230815.0 build for a few minutes while the new update was downloading, DID NOT receive this notification. 

After updating to the newest/most recent build and going through some other messages and I randomly got two notifications from Corne2Plum3, thinking they are new messages I clicked on one (getting rid of it) and the other stayed. These turned out to not be new messages as the last one was from August 24th.

How this happened/possible reproduction steps:
[1] Open lazer
[2] F8 for chat
[3] Go through multiple DMs/chat channels 
[4] Possibly an older DM will appear in the notification pop-up.

![Blushing20230908-1694188065](https://github.com/ppy/osu/assets/90729492/9e32f14a-b60d-454f-89c0-3895b228404d)

**Comment by bdach** [comment]
related: https://github.com/ppy/osu/issues/24715

**Comment by cdwcgt** [comment]
A relatively stable way of reproduction:
1. Close one of direct message channel (ensure this channel have some message)
2. Re-open direct channel which closed in 1(if you close a pm channel which with banchobot, go to banchobot's profile than click start a chat.
3. Switch to other channel before history message loaded
4. After history message loaded, you will receive notification that receive pm
